ui/AssessmentManager.py

```diff
@@ -11,10 +11,34 @@ def __init__(self, module_number, assessment, current_question):
         self._load()
 
     def _load(self, module_number: int):
-        pass
+        with open() as file:
+            data = json.loads(file)
+        question_dict = data['questions']
+        for question in question_dict:
+            if question["type"] == 'multiple_choice':
+                new_question = MultipleChoice(question['question_text'], question['question_id'],
+                                              question['question_audio'], question["explanation_text"],
+                                              question["explanation_audio"], question["image_options"],
+                                              question["selected_choices"])
+            else:
```

Comment:
let's have an `if question["type"] == 'drag_and_drop'` or something, because right now we only have two question types but it's easy to presume that in the future we can add more.